File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/SplitResult.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class SplitResult {
    public List<SplitResultDocument> Documents { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SplitResult {\n");
      sb.Append("  Documents: ").Append(Documents).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose-PDF-Cloud-SDK-for-Android/src/main/java/com/aspose/pdf/model/SplitResult.java
package com.aspose.pdf.model;

import java.util.*;
public class SplitResult {
  private List<SplitResultDocument> Documents = new ArrayList<SplitResultDocument>();
  /**
	 * getDocuments
	 * Gets List<SplitResultDocument>
	 * @return Documents
	 */
  public List<SplitResultDocument> getDocuments() {
    return Documents;
  }

	/**
	 * setDocuments
	 * Sets List<SplitResultDocument>
	 * @param Documents List<SplitResultDocument>
	 */
  public void setDocuments(List<SplitResultDocument> Documents) {
    this.Documents = Documents;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SplitResult {\n");
    sb.append("  Documents: ").append(Documents).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

